test_scripts/API/DeleteCommand/ATF_DeleteCommand_UNSUP_RES_UI_to_DelCmd_WARNINGS_to_VR_DelCmd.lua

```diff
@@ -0,0 +1,167 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS at least to one component of RPC
```

Comment:
@istoimenova , script was updated in commit a8c398943f8b88c4af198b536c410f7a945416d0